.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Show we do this weekly?

Reply:
was thinking we can see sooner how this works.. but yes, can get spammy. will switch to weekly

Reply:
Updated this..

Reply:
Thanks!